CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 ### Debugging
 
 * Enable debugger support for Zorin OS 12 ([#1160](https://github.com/OmniSharp/omnisharp-vscode/issues/1160)) _(Contributed by [@mkaziz](https://github.com/mkaziz))_
+* Added off-road support for [Windows Subsystem for Linux](https://blogs.msdn.microsoft.com/wsl/2016/04/22/windows-subsystem-for-linux-overview/) (NOTE: requires newer version of Windows than have been publicly released yet)
```

Comment:
Looks like VS Code refers 'async break' as 'pause'. So how about --

* Fixed issue with debugger pause and multithreaded call stacks (#1107 and #1105)